tests/storage_pool/storage_pool_test.go

```diff
@@ -0,0 +1,82 @@
+package tests
+
+import (
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/libopenstorage/openstorage/api"
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/gomega"
+	. "github.com/portworx/torpedo/tests"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestStoragePool(t *testing.T) {
+	RegisterFailHandler(Fail)
+	var specReporters []Reporter
+	junitReporter := reporters.NewJUnitReporter("/testresults/junit_basic.xml")
+	specReporters = append(specReporters, junitReporter)
+	RunSpecs(t, "StoragePoolResize Suite")
+}
+
+var _ = BeforeSuite(func() {
+	InitInstance()
+})
+
+var _ = Describe("{StoragePoolResize}", func() {
+
+	It("should get the existing pool", func() {
+		pools, err := Inst().V.ListStoragePools(metav1.LabelSelector{})
```

Comment:
nit: ListStoragePool() could accept "nil" to mean an empty LabelSelector for less typing

Reply:
apparently it can't